core/src/main/java/io/questdb/griffin/engine/functions/CharFunction.java

```diff
@@ -141,7 +141,7 @@ public short getShort(Record rec) {
     @Override
     public final CharSequence getStr(Record rec) {
         final char value = getChar(rec);
-        if (value == 0) {
+        if (value == Character.MIN_VALUE) {
```

Comment:
don't we use 0 as NULL for character ?

Reply:
can we make sure there is correctly named constant value for it, such as CHAR_NULL or something similar?

Reply:
I reverted this change, not needed